src/clj/game/cards/agendas.clj

```diff
@@ -251,6 +251,14 @@
     :interactive (req true)
     :effect (effect (move-zone :runner :discard :rfg))}
 
+   "City Works Project"
+   (let [meat-damage (+ 2 (:advance-counter (get-card state card) 0))]
```

Comment:
Hmmm... The `let` here is evaluated once and only once at the start of the server lifecycle. At that point there is no state, card, etc; so I'd be surprised if this actually compiled, as you're calling `+` on a function that can't evaluate because there's no variable named `state` or `card` in the given context.

The way you're using `meat-damage`, it needs to be a function that you pass `state` and `card` to from the `:msg` and `:effect` below.

Reply:
Good to know. I'll try again. Thanks for the help.